sdk/core/core-http/package.json

```diff
@@ -154,7 +154,7 @@
     "@types/chai": "^4.1.6",
     "@types/express": "^4.16.0",
     "@types/glob": "^7.1.1",
-    "@types/karma": "^3.0.0",
+    "@types/karma": "^4.4.3",
```

Comment:
I wonder why only core-http is so special as to import types for Karma? karma.conf.ts doesn't seem to import from the karma module. Can we just remove this dependency?

Reply:
I was wondering the same.  Will update PR to try removing.

Reply:
Updated PR, waiting for build results.